### DISCUSSION TITLE
Delete all completed subtasks

### DISCUSSION THREAD

**Discussion by Ceaus** [TechbeeAT/jtxBoard#1314] [Ideas]
Currently there is the possibility to globally delete all completed tasks. I'd like to also be able to delete the completed subtasks of the task I'm currently working with.

**Comment by patrickunterwegs** [comment]
Hi @Ceaus , thank you for your proposal! But is this really such a relevant use case? I mean, how many subtasks do you usually have that it would make worth it to add such a dedicated functionality?

**Reply by Ceaus** [reply]
Hi Patrick. That's easy. I don't have any toplevel tasks at all. I only have subtasks. They're all put under their own parent task to keep things tidy: stuff to buy for holidays, chores around the house, computer stuff to do, etc. 
I often find myself that I've finished a task, ticked it of, but still not wanting to delete it for context purposes.
BTW: it not super-dedicated functionality, it's already there. The only thing is that a subtask has a different (lower level) starting point. So I'd say the implementation is already 95% done  :-)

**Reply by patrickunterwegs** [reply]
Hey @Ceaus , sorry, I was caught in a bunch of work lately and forgot to follow up on here! I'll create an issue an follow up on it there =)

**Comment by Ceaus** [comment]
![Screenshot_20240409-163347_jtx_Board](https://github.com/TechbeeAT/jtxBoard/assets/13982083/a2f6c314-6016-4897-ac56-ead35dc60806)
